### DISCUSSION TITLE
How can we add custom Icon for each protocol/chain ? thx

### DISCUSSION THREAD

**Discussion by ch3ll0v3k** [wevm/wagmi#594] [Q&A]
![image](https://user-images.githubusercontent.com/7377121/174350357-fa18193f-2423-4073-8944-e5aa5903350c.png)

**Comment by ch3ll0v3k** [comment]
i have tried some options like this, but have no luck )))

![image](https://user-images.githubusercontent.com/7377121/174350650-890edaf4-7273-4be6-b000-91db3062394f.png)

**Comment by ch3ll0v3k** [comment]
found it )))

```javascript
{
  ...
  iconUrl: 'https://ftmscan.com/images/favicon.png',
  ...
}
```

**Reply by dev-cz** [reply]
How are you getting the icon to show up? All I see for chains is chain.name, chain.id, and chain.symbol?

**Reply by ch3ll0v3k** [reply]
@dev-cz  just add a line to your chain config.

iconUrl: 'https://ftmscan.com/images/favicon.png',

**Reply by dev-cz** [reply]
Oh yes I did that, but how are you pulling the icon to display. After some digging it looks like that's part of Rainbow kit. I'm only using wagmi on it's own.

**Comment by ch3ll0v3k** [comment]
![image](https://user-images.githubusercontent.com/7377121/177035544-9a2ad3b4-f4a7-4811-b41c-e4244c73b0df.png)
